### ISSUE TITLE
nn.CrossEntropyLoss should work with logit scores

### ISSUE THREAD

**Issue by coderSkyChen** [jfzhang95/pytorch-video-recognition#7]
if you use nn.CrossEntropyLoss the input should be the scores before softmax layer. So you should remove the softmax layer.

**Comment by jfzhang95** [comment]
Hi, 
Three models implemented in this repo did not contain softmax layer.
